exporter/datadogexporter/internal/translator/metrics_translator_test.go

```diff
@@ -471,18 +479,106 @@ func TestMapHistogramMetrics(t *testing.T) {
 	}
 
 	tr.cfg.Buckets = false
+	res, sl = tr.mapHistogramMetrics("doubleHist.test", slice, []string{"attribute_tag:attribute_value"})
+	require.Empty(t, sl)
 	assert.ElementsMatch(t,
-		tr.mapHistogramMetrics("doubleHist.test", slice, []string{"attribute_tag:attribute_value"}), // No buckets
+		res, // No buckets
 		noBucketsAttributeTags,
 	)
 
 	tr.cfg.Buckets = true
+	tr.cfg.BucketsAsCounts = true
+	res, sl = tr.mapHistogramMetrics("doubleHist.test", slice, []string{"attribute_tag:attribute_value"})
+	require.Empty(t, sl)
 	assert.ElementsMatch(t,
-		tr.mapHistogramMetrics("doubleHist.test", slice, []string{"attribute_tag:attribute_value"}), // buckets
+		res, // buckets
 		append(noBucketsAttributeTags, bucketsAttributeTags...),
 	)
 }
 
+func TestHistogramSketches(t *testing.T) {
+	N := 1_000
+	M := 20_000.0
+
+	// Given a probability distribution function for a distribution
+	// with support [0, N], generate an OTLP Histogram data point with N buckets,
+	// (-inf, 0], (0, 1], ..., (N-1, N], (N, inf)
+	// which contains N*M uniform samples of the distribution.
+	fromPDF := func(pdf func(x float64) float64) pdata.HistogramDataPoint {
+		p := pdata.NewHistogramDataPoint()
+		bounds := make([]float64, N+1)
+		buckets := make([]uint64, N+2)
+		buckets[0] = 0
+		for i := 0; i <= N; i++ {
+			bounds[i] = float64(i)
+			// the bucket with bounds (i, i+1) has the value
+			// of the pdf in the midpoint of the bucket (i + .5)
+			buckets[i+1] = uint64(pdf((float64(i) + 0.5)) * M)
```

Comment:
I think you could use the `cdf` here instead:
```suggestion
			buckets[i+1] = uint64((cdf(bounds[i+1]) - cdf(bounds[i])) * M)
```
Formally, that should be the sum of the `pdf` over the bucket (rather than the value of the pdf in the midpoint), which is what the `cdf` difference above is equal to.

Reply:
I addressed this, PTAL @CharlesMasson